bsd.go

```diff
@@ -7,23 +7,23 @@ package gopass
 #include <unistd.h>
 #include <stdio.h>
 
-int getch() {
-        int ch;
+char getch(int termDescriptor) {
+        char ch;
         struct termios t_old, t_new;
 
-        tcgetattr(STDIN_FILENO, &t_old);
+        tcgetattr(termDescriptor, &t_old);
         t_new = t_old;
         t_new.c_lflag &= ~(ICANON | ECHO);
-        tcsetattr(STDIN_FILENO, TCSANOW, &t_new);
+        tcsetattr(termDescriptor, TCSANOW, &t_new);
 
-        ch = getchar();
+        read(termDescriptor, &ch, sizeof(ch));
```

Comment:
The problem with `getchar` is that it's implicitly reading from stdin, instead of from the file handle we want to pass it. What I want to do is "read a single byte from a file descriptor in the Most Standard Way." Do you know what way that is?